osCam/static/core/dash.css

```diff
@@ -13,6 +13,20 @@ hr {
     border: none;
 }
 
+
+/* For Video Container (Move to its own css file later) */
```

Comment:
do you still need to move to it's own css file? is it later now? or should we expect this in the future?

Reply:
Ya we should have a dir for each app's static content

Reply:
its moved already just doesn't show here yet